src/test/java/com/google/javascript/clutz/aliased_enums.d.ts

```diff
@@ -1,14 +1,14 @@
 declare namespace ಠ_ಠ.clutz_internal.nested.bar {
-  type Enum = number ;
-  var Enum : {
+  type HahaEnum = number ;
+  var HahaEnum : {
     A : ಠ_ಠ.clutz_internal.nested.baz.Enum ,
```

Comment:
Yeah, this was conscious on my side. But it turns out we don't have to explicitly emit the `getMembersType` value, as we obviously just aliased the correct type. So I'm using the unqualified name now.

Re enum only works for same name, you might also have run into accidentally aliasing the type at the time.